cdk/visit/lambda_code/log_visit.py

```diff
@@ -130,6 +132,7 @@ def handler(request, context):
     try:
         # Get the username from the request body.
         username = json.loads(request["body"])["username"]
+        location = json.loads(request["body"])["location"]
```

Comment:
We can't merge this in yet because the front-end won't be submitting requests that contain the `location` param. Merging this code would cause the requests to start throwing a `KeyError` during this `json.loads()`.

Perhaps you should wrap this in a try-catch, or update your PR to merge this into a feature branch instead of mainline.

Reply:
Got it! Wrapped it in one of my lower commits